server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -771,23 +858,29 @@ XCTL : X C T L;
 XML : X M L;
 XMLPARSE : X M L P A R S E;
 XMLSS : X M L S S;
-XOPTS: X O P T S;
 XP : X P;
 XREF : X R E F;
 X_CHAR : X;
 YEAR : Y E A R;
-YEARWINDOW : Y E A R W I N D O W;
 YES : Y E S;
-YW : Y W;
 YYYYDDD : Y Y Y Y D D D;
 YYYYMMDD : Y Y Y Y M M D D;
+ZC : Z C;
+ZD : Z D;
 ZERO : Z E R O;
 ZEROES : Z E R O E S;
 ZEROS : Z E R O S;
+ZLEN : Z L E N;
+ZON : Z O N;
+ZONECHECK : Z O N E C H E C K;
+ZONEDATA : Z O N E D A T A;
 ZWB : Z W B;
 
 UNDERSCORECHAR : '_';
 
+INTEGERLITERAL_WITH_K: INTEGERLITERAL ('K' | 'k');
+CURRENCY_SYMBOL : [\p{Sc}];
```

Comment:
Please, add a test for the symbol and provide a replaceable token for it in `SpecialTokenHandling.properties`

Reply:
I've done those tests as a part of the UseCases set.
INTEGERLITERAL_WITH_K : https://github.com/grianbrcom/che-che4z-lsp-for-cobol/blob/compilerOptions/server/src/test/java/org/eclipse/lsp/cobol/usecases/TestProcessCbl.java#L77
CURRENCY_SYMBOL : https://github.com/grianbrcom/che-che4z-lsp-for-cobol/blob/compilerOptions/server/src/test/java/org/eclipse/lsp/cobol/usecases/TestProcessCbl.java#L108

Reply:
But it contains only a positive case. What will happen if I put `CURR(foo)`?